lib/app/widgets/main_layout.dart

```diff
@@ -0,0 +1,16 @@
+import 'package:flutter/material.dart';
+import 'package:gruene_app/app/widgets/bottom_navigation.dart';
+
+class MainLayout extends StatelessWidget {
```

Comment:
:upside_down_face: Perhaps:
```suggestion
class HomeScreen extends StatelessWidget {
```

Reply:
I don't understand this suggestion. This layout is used on all screens...?

Reply:
Sorry, I missed that. Thanks for explaining.